xtask/Cargo.toml

```diff
@@ -7,11 +7,11 @@ edition = "2021"
 publish = false
 
 [dependencies]
+flate2 = { version = "1" }
```

Comment:
NIT: Preserve alphabetic ordering of crates.

Reply:
eb881ab